packages.config

```diff
@@ -1,8 +1,8 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="MSBuildTasks" version="1.5.0.196" targetFramework="net20" developmentDependency="true" />
   <package id="StyleCop.Analyzers" version="1.0.0" targetFramework="net20" developmentDependency="true" />
-  <package id="xunit" version="1.9.2" targetFramework="net20" developmentDependency="true" />
-  <package id="xunit.runners" version="1.9.2" targetFramework="net20" developmentDependency="true" />
-  <package id="xunit.runner.visualstudio" version="2.2.0" targetFramework="net20" developmentDependency="true" />
-</packages>
+  <package id="xunit" version="1.9.2" targetFramework="net20" />
+  <package id="xunit.runner.visualstudio" version="2.3.0" targetFramework="net20" developmentDependency="true" />
+  <package id="xunit.runners" version="1.9.2" targetFramework="net20" />
```

Comment:
These lines have lost their developmentDependency="true" attributes.

Reply:
Assuming xunit made those changes when i uninstall/reinstalled/updated it.
Do i direct edit the file and add the developmentDependency statement?